app/src/androidTest/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -136,6 +137,22 @@ open class BpkSnapshotTest {
     }
   }
 
+  private fun View.findComposeView(): ComposeView? {
+    if (this !is ViewGroup) return null
+    for (i in 0..childCount) {
+      val child = getChildAt(i)
+      if (child is ComposeView) {
+        return child
+      } else if (child is ViewGroup) {
+        val view = child.findComposeView()
+        if (view != null) {
```

Comment:
I think you can just return child.findComposeView() since you have the check and return case at the beginning of the method... no?

Reply:
Not quite - this will return null on the first child if it's not a compose view, so it won't go to the other children to check